contracts/Rewards/RewardsDistributor.sol

```diff
@@ -226,14 +234,40 @@ contract RewardsDistributor is ExponentialNoError, Ownable2StepUpgradeable, Acce
         uint256 numTokens = vTokens.length;
         require(
             numTokens == supplySpeeds.length && numTokens == borrowSpeeds.length,
-            "RewardsDistributor::setRewardTokenSpeeds invalid input"
+            "setRewardTokenSpeeds invalid input"
```

Comment:
Why do we remove the reference of contract from here as this is not consistence over the contract see [here](https://github.com/VenusProtocol/isolated-pools/pull/257/files#diff-f2ada847b308c6b0c4d97a6f4e4020042c3fec32b83b4021b496a34d42b5332dR260)